roles/timer/tasks/main.yml

```
@@ -0,0 +1,24 @@
+---
+- name: Fail when timer_name is empty
+  fail: msg="timer_name is empty"
+  when: not timer_name
+
+- name: Upload service and timer
+  template:
+    src: "job.{{ item }}"
+    dest: /lib/systemd/system/{{ timer_name }}.{{ item }}
+  with_items:
+  - service
+  - timer
+  register: template_result
```

```suggestion
  notify: reload systemd
```

and something in `roles/timer/handlers/main.yml`